tests/cases/test_raft.rs

```diff
@@ -425,6 +428,7 @@ impl Network {
 
 #[test]
 fn test_progress_become_probe() {
+    let _ = env_logger::try_init();
```

Comment:
can we use one place to init the logger? 
like TiKV CI does https://github.com/tikv/tikv/blob/master/tests/integrations.rs#L63

Reply:
In a CI you run a full test suite, this is targeted at users trying to trace/debug/log a single or regex'd test.

Once #52 is merged I think we should write the logs of each test to a folder and we can display them nicely when they fail.

Reply:
@Hoverbear 

I don't think it is a good style to init logger everywhere. it looks weird.

Reply:
While I agree, it is the suggested style: https://github.com/sebasmagri/env_logger#in-tests

Regardless, there's no way **not** to have to init the logger ever test without relying on some test always being run first or something, which breaks `cargo test $TEST` workflows.